test/public/defaults.js

```diff
@@ -439,3 +439,14 @@ module.exports.checkMismatchingUrlParam = async (page, expectedUrlParameters) =>
     }
     return ret;
 };
+
+/**
+ * Wait For table to reload data
```

Comment:
Please try to make effort in your docblock.
`Call a trigger function, wait for the table to display a loading spinner then wait for the loading spinner to be removed`

Reply:
I would simply remove the first line too

Reply:
applied